### ISSUE TITLE
FSE failing to load on simple site with custom domain

### ISSUE THREAD

**Issue by SiobhyB** [Automattic/wp-calypso#76109]
### Quick summary

After switching the theme of my personal site to [the Byrne theme](PdyWFn-9-p2) this weekend, I attempted to edit the site via the FSE. However, the editor does not load: 

<img width="1718" alt="Screenshot 2023-04-23 at 19 18 40" src="https://user-images.githubusercontent.com/2998162/233857664-94090449-fe2d-4089-8efe-3d7833e078fd.png">

The relevant console errors are as follows:

```
- Store "core/interface" is already registered.
- Store "automattic/site" is already registered.
- Access to fetch at 'https://siobhan.blog/?_wp-find-template=true' from origin 'https://siobhanbamber.wordpress.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
- Failed to load resource: net::ERR_FAILED
```

The site this is happening on is a simple WordPress.com site that _does not_ have Gutenberg edge enabled. 

I found a similar mention of a broken site editor with a CORS error at pddgU7-8H-p2#comment-250. cc-ing @alshakero, who was involved in that discussion, in case it's relevant.

I also found a reference to the `Store "automattic/site" is already registered` error in https://github.com/Automattic/wp-calypso/issues/75179. However, that report is limited to sites with Gutenberg edge enabled and does not cause the FSE to stop working.

### Steps to reproduce

I have tried (and failed) to reproduce the error on different sites with the following set ups:

- A separate simple site with a custom `.blog` domain and the Byrne theme.
- A simple site _without_ a custom domain and the Byrne theme (tested with Gutenberg edge both enabled and disabled).
- A simple site with a custom `.wales` domain (tested with both the Byrne and the Verso themes).

The site editor loaded as expected for me in the above scenarios. It's only failing on my own personal site at siobhan.blog.

### What you expected to happen

It should be possible to preview and edit my site via the FSE.

### What actually happened

A perpetual progress bar is displayed instead of the FSE. I was able to eventually make the edits I wanted by navigating through to the individual _Templates_ and _Template Parts_, but this is unintuitive and most users would not know how to do this.

I understand that this report may be considered low priority given the lack of reproducible steps, but I wanted to make sure to log this report anyway, given this might end up blocking other users.

### Impact

One

### Available workarounds?

Yes, difficult to implement

### Platform (Simple and/or Atomic)

Simple

### Logs or notes

Tested on Google Chrome version `112.0.5615.137`.

**Comment by cuemarie** [comment]
Closing as duplicate of https://github.com/Automattic/wp-calypso/issues/76105
